quadrotor_aerodynamics:
        C_wxy: 0.08
        C_wz: 0.08
        C_mxy: 0.074156208000000
        C_mz: 0.050643264000000
